### ISSUE TITLE
Broadcast audio only: hide video window, leave play/mute toggle button only

### ISSUE THREAD

**Issue by dprisikar** [AirenSoft/OvenPlayer#282]
Hi, 

I am using a player for sending audio-only to a listener, and need to hide the video window. 

Assuming that, should such option exist, it should be listed here: https://airensoft.gitbook.io/ovenplayer/initialization, unfortunately, I did not find such option on this page. 

I do realize that one can adjust <div class="player-wrapper"> by controlling the class CSS properties, but it's not taking me anywhere farther than changing the video window's width and height, rather than hiding it all the way, with a play button still being visible.

Overall, what I am trying to achieve here is a user being able to join a live audio-only stream, by tapping a play button. A nice to have option is making this play button toggle to "mute", after the user has tapped "play" and has joined the audio stream, so that the user could stop listening to the stream without leaving the page. 

Using plain vanilla Javascript (+ CSS/HTML). 

Any code snippets with a working solution/relevant examples would be very much appreciated.

Thank you for providing timely help in advance!

**Comment by IanMitchell77** [comment]
I have not coded this but surely you just hide the player and draw HTML buttons that call the OvenPlayer's API?

**Comment by dprisikar** [comment]
@IanMitchell77, thanks for your comment!

You mean, I draw new buttons and call the functions listed in the API (https://airensoft.gitbook.io/ovenplayer/api-reference/api) with the buttons, e.g. ...onclick=playerInstance.play() ?

**Comment by IanMitchell77** [comment]
Exactly - the buttons being plain old HTML or whatever but they call the JavaScript API to the player like you've linked to.

The stream that you play does not have to have a video element so you could have an audio only stream but maybe your already doing that.

**Comment by dprisikar** [comment]
Right! Trying that. And thanks again for your reply, @IanMitchell77 

Here is the code:

<div class="form-floating mb-4 mx-lg-5 px-5">
 
<div class="player-wrapper">
  
  <div id="player_id"></div>
</div>
 </div>

<div class="form-floating mb-4">
<input type="button" class="button-publish" id="muteBtn" value="PLAY" onclick="playerInstance.play()">
</div>

======
Receiving "Uncaught ReferenceError: playerInstance is not defined"

Any thoughts on how to make the reference work?

**Comment by dprisikar** [comment]
Yeah, right! So obvious 😅

Thanks again, @IanMitchell77

**Comment by dprisikar** [comment]
Any chance you could take a very quick look at this other issue, @IanMitchell77 ? 

Trying to mute speaker's mic with DOM, but not quite sure how to access the audio stream.

https://github.com/AirenSoft/OvenLiveKit-Web/issues/11
